lib/blueprintLocations.js

```diff
@@ -1,4 +1,4 @@
-const {hash, parseChance} = require("./utils.js")
+const {hash, addUniqueHashedObject, parseChance} = require("./utils.js")
```

Comment:
Add the new one at the end of the destructuring. Objects are not sorted, and if you're going for alphabetical, put it at the beginning.

Reply:
I'm not. This is the same order as `utils.js`.

Also, it made sense to put it after the hash function more than alphabetical because shouldn't functions be defined before being called?

Reply:
> Also, it made sense to put it after the hash function more than alphabetical because shouldn't functions be defined before being called?

You define & declare them with this statement and the order doesn't matter even if one of them uses another one internally (because they already have a reference from their own file). I don't really see a reason why you put it in the center though, either at the end for hey new thing or at the front because then it's alphabetically sorted :>.